menutoplinks.class.php

```diff
@@ -57,17 +57,19 @@ public static function getLinkLang($id_linksmenutop, $id_shop)
         $link = array();
         $label = array();
         $new_window = false;
+        $new_popup = false;
 
         foreach ($ret as $line) {
             $link[$line['id_lang']] = Tools::safeOutput($line['link']);
             $label[$line['id_lang']] = Tools::safeOutput($line['label']);
             $new_window = (bool)$line['new_window'];
+            $new_popup = (bool)$line['new_popup'];
         }
 
-        return array('link' => $link, 'label' => $label, 'new_window' => $new_window);
+        return array('link' => $link, 'label' => $label, 'new_window' => $new_window, 'new_popup' => $new_popup);
     }
 
-    public static function add($link, $label, $newWindow = 0, $id_shop)
+    public static function add($link, $label, $newWindow = 0, $newPopup = 0, $id_shop)
```

Comment:
Hi,

One can have overrided it, I understand your point but for the moment you need to add params at the end.

Regards